app/assets/stylesheets/modules/homepage.scss

```diff
@@ -0,0 +1,44 @@
+@import "variables";
+@import "mixins";
+
+body.home-page {
+  background: url(dog.png) no-repeat;
+  background-size: 100%;
+  opacity: 0.9;
+
+  .right {
+    padding: 1em;
+    padding-left: 2em;
+
+    a {
+      font-family: "Poiret One";
+      color: #ffffff;
```

Comment:
Color literals like `#ffffff` should only be used in variable declarations; they should be referred to via variable everywhere else.